Modules/_tracemalloc.c

```diff
@@ -1643,6 +1643,28 @@ _tracemalloc_get_traced_memory_impl(PyObject *module)
     return Py_BuildValue("nn", size, peak_size);
 }
 
+/*[clinic input]
+_tracemalloc.reset_peak
+
+Set the peak size of memory blocks traced by tracemalloc to the current size.
+
```

Comment:
I suggest to add (here and in the doc):

```
Do nothing if the tracemalloc module is not tracing memory allocations.
```

In tracemalloc.rst, you can use:

```
:mod:`tracemalloc`
```

Reply:
I've done so. 👍

Reply:
Thanks.